pkg/render/renderer_impl.go

```diff
@@ -333,12 +332,14 @@ func templateFuncs() htmltemplate.FuncMap {
 	}
 }
 
-func hasOne(a []*interface{}) bool {
-	return len(a) == 1
+func hasOne(a interface{}) bool {
+	s := reflect.ValueOf(a)
+	return s.Len() == 1
```

Comment:
Len will panic on non-array/slice/string types. Isn't reflection like the most funest?